controllers/client/user.controller.js

```diff
@@ -27,3 +27,42 @@ module.exports.registerPost = async (req,res)=>{
     res.cookie("tokenUser" , user.tokenUser);
     res.redirect("/");
 }
+//[Get]/user/login
+module.exports.login = async(req, res)=>{
+    res.render("client/pages/user/login",{
+        pageTitle: "Trang Đăng Nhập"
+    })
+}
+//[Post]/user/loginPost
+module.exports.loginPost = async(req, res)=>{
+    const email = req.body.email;
+    const password = req.body.password;
+    const user = await User.findOne({
+        email: email,
+        deleted:false
+    });
+    if(!user){
+        req.flash("error" ,"Email Không Tồn Tại!");
+        res.redirect("back");
+        return;
+    }
+    const userPassword = user.password;
+    const check = await bcrypt.compare(password,userPassword);
```

Comment:
```suggestion
    const check = await bcrypt.compare(password, user.password);
```

no need to make an extra variable here. there's no benefit in doing so.